dashr/chapters/dashDataTable/part4/index.R

```diff
@@ -0,0 +1,133 @@
+  filteringSortingwithGraph = utils$LoadExampleCode('dashr/chapters/dashDataTable/part4/examples/filteringSortingwithGraph.R')
+)
+
+layout <- htmlDiv(
+  list(
+    dccMarkdown("
+# DataTable - R Callbacks
+
+## Backend Paging
+
+With backend paging, we can load data into our table progressively. 
+Instead of loading all of the data at once, 
+we'll only load data as the user requests it 
+when they click on the \"previous\" and \"next\" buttons.
+
+Since backend paging integrates directly with your Dash callbacks, 
+you can load your data from any R data source.
+              "),
+    
+    examples$backendPaging$source_code,
+    examples$backendPaging$layout,
+    
+    htmlHr(),
+    
+    dccMarkdown("
+With backend paging, we can have front-end sorting and filtering 
+but it will only filter and sort the data that exists on the page.
+
+This should be avoided. 
+Your users will expect that sorting and filtering is 
+happening on the entire dataset and, with large pages, 
+might not be aware that this is only occuring on the current page.
+              
+Instead, we recommend implmenting sorting and filtering on the backend as well. 
+That is, on the entire underlying dataset.
+              
+**Note for returning users - changed property names:**
+              
+- Sorted fields are now in `sort_by`, not `sorting_settings`
+
+- The filter string is now in `filter`, not `filtering_settings`
+
+## Backend Paging with Sorting
+              "),
+    
+    examples$backendPagingWithSorting$source_code,
+    examples$backendPagingWithSorting$layout,
+    
+    
+    dccMarkdown("
+## Backend Paging with Multi Column Sorting
+
+Multi-column sort allows you to sort by multiple columns. 
+This is useful when you have categorical columns with repeated values and 
+you're interested in seeing the sorted values for each category.
+
+In this example, try sorting by continent and then any other column.
+              "),
+    
+    examples$backendPagingWithMultiSorting$source_code,
+    examples$backendPagingWithMultiSorting$layout,
+    
+    dccMarkdown("
+## Backend Paging with Filtering
+
+DataTable's front-end filtering has its own filtering expression language.
+
+Currently, backend filtering must parse the same filtering language. 
+If you write an expression that is not \"valid\" under the filtering language, 
+then it will not be passed to the backend.
+              
+This limitation will be removed in the future to allow you to 
+write your own expression query language.
+
+In this example, we've written a Pandas backend for the filtering language. 
+It supports `eq`, `<`, and `>`. For example, try:
+
+- Enter `eq Asia` in the \"continent\" column
+
+- Enter `> 5000` in the \"gdpPercap\" column
+
+- Enter `< 80` in the \"lifeExp\" column
+
+> Note that unlike the front-end filtering, 
+> our backend filtering expression language doesn't require or support `num()` or 
```

Comment:
```suggestion
> our back-end filtering expression language doesn't require or support `num()` or 
```

We should use `front-end` and `back-end` or `frontend` and `backend` in the interest of consistency.

Reply:
Right, we should use either `front-end` and `back-end` or `frontend` and `backend`